src/com/palmergames/bukkit/towny/object/Town.java

```diff
@@ -434,6 +434,9 @@ public double getTownBlockCostN(int inputN) throws TownyException {
 		double nextprice = getTownBlockCost();
 		int i = 1;
 		double cost = nextprice;
+		if(TownySettings.getMaxClaimPrice() != -1)
+			cost *=  inputN;
+		else 
```

Comment:
If you could use some { }'s and indent the while I think this will be good to go.

Reply:
Nevermind, I see you've changed it around a bit.